src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,466 @@
+    <p><small>By Dan Dascalescu<br>
+      Published: June 12, 2016</p></small></p>
+  </div>
+
+  <!-- #### Social Sharing  -->
+
+  <!--
+  The Social Share extension provides a common interface for share buttons.  Learn more about [`amp-social-share`](/components/amp-social-share/).
+  -->
+  <p class="heading">
+    <amp-social-share type="twitter" data-param-text="Amazing recipe that my friend made" width="45" height="33"></amp-social-share>
+    <amp-social-share type="facebook" width="45" height="33" data-attribution=254325784911610></amp-social-share>
+    <amp-social-share type="gplus" width="45" height="33"></amp-social-share>
+    <amp-social-share type="email" width="45" height="33"></amp-social-share>
+    <amp-social-share type="pinterest" width="45" height="33"></amp-social-share>
+  </p>
+
+  <!-- #### Image Galleries  -->
+  <!--
+  Use `amp-carousel` for image galleries. Here we display responsively images with different aspect ratios. Learn more about [creating image galleries in AMP](/advanced/image_galleries_with_amp-carousel/). The `loop` attribute lets us start with a picture of the finished product, and get to the same finished when advancing to the slide after the picture of the dish just before putting it in the oven. What You Start With Is What You Get.
+  -->
+  <amp-carousel width="1280" height="970" layout="responsive" type="slides" loop>
+    <figure>
+      <amp-img src="/img/lamb-garlic-done.jpg" layout="fill"></amp-img>
+      <figcaption>Serve with a side of baked potatoes.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/meat-silverskin-cutting.jpg" layout="fill"></amp-img>
+      <figcaption>Prepare the meat by washing and removing the silverskin.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/garlic.jpg" layout="fill"></amp-img>
+      <figcaption>Prepare the garlic.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/lamb-with-garlic-slivers.jpg" layout="fill"></amp-img>
+      <figcaption>Insert slivers of garlic into the meat; the deeper the better.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/tomatoes-cut.jpg" layout="fill"></amp-img>
+      <figcaption>Cut the tomatoes.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/lamb-garlic-ready-for-oven.jpg" layout="fill"></amp-img>
+      <figcaption>The garlic lamb roast is ready for baking.</figcaption>
+    </figure>
+  </amp-carousel>
+
+
+  <p>This is a traditional Romanian lamb recipe, with the subtle taste of garlic deeply permeating the meat.</p>
+
+  <!-- We should switch to the [AMP star rating component](https://github.com/ampproject/amphtml/issues/2691) when it becomes available. -->
+  <p><div class="star-rating" style="display: inline-block; vertical-align: middle">
```

Comment:
All styles must live in the head of the document, I think this is making the CI fails.